src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/multiples/MultiplesSendNotificationService.java

```diff
@@ -55,6 +65,18 @@ public void setHearingDetailsFromLeadCase(MultipleDetails multipleDetails, List<
         }
     }
 
+    public void setMultipleWithExcelFileData(MultipleDetails multipleDetails, String userToken, List<String> errors) {
+        multipleDynamicListFlagsService.populateDynamicListFlagsLogic(userToken, multipleDetails, errors);
+        if (errors.isEmpty()) {
+            String caseTypeId = multipleDetails.getCaseTypeId();
+            if (ENGLANDWALES_BULK_CASE_TYPE_ID.equals(caseTypeId)) {
+                fileLocationSelectionService.initialiseFileLocation(multipleDetails.getCaseData());
+            } else if (SCOTLAND_BULK_CASE_TYPE_ID.equals(caseTypeId)) {
+                scotlandFileLocationSelectionService.initialiseFileLocation(multipleDetails.getCaseData());
+            }
```

Comment:
Is it worth adding some sort of error if the case type is unexpected? Perhaps just an error log message since its probably not something the end user would be in control of

Reply:
Probably an unlikely scenario but I've added it. If it did happen I think best to return the error than just log it. Stop the user from going any further/ force them to restart the process.